Pylint: W0102

```bash
mmpose/models/backbones/hrnet.py:19:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/hrnet.py:264:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/shufflenet_v1.py:45:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/shufflenet_v1.py:45:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/shufflenet_v1.py:177:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/shufflenet_v1.py:177:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/cpm.py:71:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/regnet.py:84:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/hourglass.py:26:4: ```: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/hourglass.py:115:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/resnet.py:34:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/resnet.py:149:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/resnet.py:327:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/resnet.py:477:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/mobilenet_v3.py:65:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/mobilenet_v2.py:32:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/mobilenet_v2.py:32:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/mobilenet_v2.py:126:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/mobilenet_v2.py:126:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/shufflenet_v2.py:31:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/shufflenet_v2.py:31:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/shufflenet_v2.py:153:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/shufflenet_v2.py:153:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/scnet.py:25:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/utils/inverted_residual.py:36:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
mmpose/models/backbones/utils/inverted_residual.py:36:4: W0102: Dangerous default value dict() (builtins.dict) as argument (dangerous-default-value)
```

--- a/mmpose/models/backbones/cpm.py
+++ b/mmpose/models/backbones/cpm.py
@@ -77,6 +77,7 @@ class CPM(BaseBackbone):
                  middle_channels=32,
                  num_stages=6,
                  norm_cfg=dict(type='BN', requires_grad=True)):
+        # Protect mutable default arguments
         norm_cfg = copy.deepcopy(norm_cfg)
         super().__init__()
 

--- a/mmpose/models/backbones/cpm.py
+++ b/mmpose/models/backbones/cpm.py
@@ -1,3 +1,5 @@
+import copy
+
 import torch
 import torch.nn as nn
 from mmcv.cnn import ConvModule, constant_init, normal_init
@@ -75,6 +77,7 @@ class CPM(BaseBackbone):
                  middle_channels=32,
                  num_stages=6,
                  norm_cfg=dict(type='BN', requires_grad=True)):
+        norm_cfg = copy.deepcopy(norm_cfg)
         super().__init__()
 
         assert in_channels == 3

--- a/mmpose/models/backbones/hourglass.py
+++ b/mmpose/models/backbones/hourglass.py
@@ -1,3 +1,5 @@
+import copy
+
 import torch.nn as nn
 from mmcv.cnn import ConvModule, constant_init, normal_init
 from torch.nn.modules.batchnorm import _BatchNorm
@@ -28,6 +30,7 @@ class HourglassModule(nn.Module):
                  stage_channels,
                  stage_blocks,
                  norm_cfg=dict(type='BN', requires_grad=True)):
+        norm_cfg = copy.deepcopy(norm_cfg)
         super().__init__()
 
         self.depth = depth
@@ -119,6 +122,7 @@ class HourglassNet(BaseBackbone):
                  stage_blocks=(2, 2, 2, 2, 2, 4),
                  feat_channel=256,
                  norm_cfg=dict(type='BN', requires_grad=True)):
+        norm_cfg = copy.deepcopy(norm_cfg)
         super().__init__()
 
         self.num_stacks = num_stacks

--- a/mmpose/models/backbones/hourglass.py
+++ b/mmpose/models/backbones/hourglass.py
@@ -30,6 +30,7 @@ class HourglassModule(nn.Module):
                  stage_channels,
                  stage_blocks,
                  norm_cfg=dict(type='BN', requires_grad=True)):
+        # Protect mutable default arguments
         norm_cfg = copy.deepcopy(norm_cfg)
         super().__init__()
 
@@ -122,6 +123,7 @@ class HourglassNet(BaseBackbone):
                  stage_blocks=(2, 2, 2, 2, 2, 4),
                  feat_channel=256,
                  norm_cfg=dict(type='BN', requires_grad=True)):
+        # Protect mutable default arguments
         norm_cfg = copy.deepcopy(norm_cfg)
         super().__init__()
 

--- a/mmpose/models/backbones/hrnet.py
+++ b/mmpose/models/backbones/hrnet.py
@@ -28,6 +28,8 @@ class HRModule(nn.Module):
                  with_cp=False,
                  conv_cfg=None,
                  norm_cfg=dict(type='BN')):
+
+        # Protect mutable default arguments
         norm_cfg = copy.deepcopy(norm_cfg)
         super().__init__()
         self._check_branches(num_branches, num_blocks, in_channels,
@@ -272,6 +274,7 @@ class HRNet(nn.Module):
                  norm_eval=False,
                  with_cp=False,
                  zero_init_residual=False):
+        # Protect mutable default arguments
         norm_cfg = copy.deepcopy(norm_cfg)
         super().__init__()
         self.extra = extra

--- a/mmpose/models/backbones/hrnet.py
+++ b/mmpose/models/backbones/hrnet.py
@@ -1,3 +1,5 @@
+import copy
+
 import torch.nn as nn
 from mmcv.cnn import (build_conv_layer, build_norm_layer, constant_init,
                       normal_init)
@@ -26,6 +28,7 @@ class HRModule(nn.Module):
                  with_cp=False,
                  conv_cfg=None,
                  norm_cfg=dict(type='BN')):
+        norm_cfg = copy.deepcopy(norm_cfg)
         super().__init__()
         self._check_branches(num_branches, num_blocks, in_channels,
                              num_channels)
@@ -269,6 +272,7 @@ class HRNet(nn.Module):
                  norm_eval=False,
                  with_cp=False,
                  zero_init_residual=False):
+        norm_cfg = copy.deepcopy(norm_cfg)
         super().__init__()
         self.extra = extra
         self.conv_cfg = conv_cfg

--- a/mmpose/models/backbones/mobilenet_v2.py
+++ b/mmpose/models/backbones/mobilenet_v2.py
@@ -39,6 +39,7 @@ class InvertedResidual(nn.Module):
                  norm_cfg=dict(type='BN'),
                  act_cfg=dict(type='ReLU6'),
                  with_cp=False):
+        # Protect mutable default arguments
         norm_cfg = copy.deepcopy(norm_cfg)
         act_cfg = copy.deepcopy(act_cfg)
         super().__init__()
@@ -135,6 +136,7 @@ class MobileNetV2(BaseBackbone):
                  act_cfg=dict(type='ReLU6'),
                  norm_eval=False,
                  with_cp=False):
+        # Protect mutable default arguments
         norm_cfg = copy.deepcopy(norm_cfg)
         act_cfg = copy.deepcopy(act_cfg)
         super().__init__()

--- a/mmpose/models/backbones/mobilenet_v2.py
+++ b/mmpose/models/backbones/mobilenet_v2.py
@@ -1,3 +1,4 @@
+import copy
 import logging
 
 import torch.nn as nn
@@ -38,6 +39,8 @@ class InvertedResidual(nn.Module):
                  norm_cfg=dict(type='BN'),
                  act_cfg=dict(type='ReLU6'),
                  with_cp=False):
+        norm_cfg = copy.deepcopy(norm_cfg)
+        act_cfg = copy.deepcopy(act_cfg)
         super().__init__()
         self.stride = stride
         assert stride in [1, 2], f'stride must in [1, 2]. ' \
@@ -132,6 +135,8 @@ class MobileNetV2(BaseBackbone):
                  act_cfg=dict(type='ReLU6'),
                  norm_eval=False,
                  with_cp=False):
+        norm_cfg = copy.deepcopy(norm_cfg)
+        act_cfg = copy.deepcopy(act_cfg)
         super().__init__()
         self.widen_factor = widen_factor
         self.out_indices = out_indices

--- a/mmpose/models/backbones/mobilenet_v3.py
+++ b/mmpose/models/backbones/mobilenet_v3.py
@@ -1,3 +1,4 @@
+import copy
 import logging
 
 import torch.nn as nn
@@ -70,6 +71,7 @@ class MobileNetV3(BaseBackbone):
                  frozen_stages=-1,
                  norm_eval=False,
                  with_cp=False):
+        norm_cfg = copy.deepcopy(norm_cfg)
         super().__init__()
         assert arch in self.arch_settings
         for index in out_indices:

--- a/mmpose/models/backbones/mobilenet_v3.py
+++ b/mmpose/models/backbones/mobilenet_v3.py
@@ -71,6 +71,7 @@ class MobileNetV3(BaseBackbone):
                  frozen_stages=-1,
                  norm_eval=False,
                  with_cp=False):
+        # Protect mutable default arguments
         norm_cfg = copy.deepcopy(norm_cfg)
         super().__init__()
         assert arch in self.arch_settings

--- a/mmpose/models/backbones/regnet.py
+++ b/mmpose/models/backbones/regnet.py
@@ -100,6 +100,7 @@ class RegNet(ResNet):
                  norm_eval=False,
                  with_cp=False,
                  zero_init_residual=True):
+        # Protect mutable default arguments
         norm_cfg = copy.deepcopy(norm_cfg)
         super(ResNet, self).__init__()
 

--- a/mmpose/models/backbones/regnet.py
+++ b/mmpose/models/backbones/regnet.py
@@ -1,3 +1,5 @@
+import copy
+
 import numpy as np
 import torch.nn as nn
 from mmcv.cnn import build_conv_layer, build_norm_layer
@@ -98,6 +100,7 @@ class RegNet(ResNet):
                  norm_eval=False,
                  with_cp=False,
                  zero_init_residual=True):
+        norm_cfg = copy.deepcopy(norm_cfg)
         super(ResNet, self).__init__()
 
         # Generate RegNet parameters first

--- a/mmpose/models/backbones/resnet.py
+++ b/mmpose/models/backbones/resnet.py
@@ -1,3 +1,5 @@
+import copy
+
 import torch.nn as nn
 import torch.utils.checkpoint as cp
 from mmcv.cnn import (ConvModule, build_conv_layer, build_norm_layer,
@@ -42,6 +44,7 @@ class BasicBlock(nn.Module):
                  with_cp=False,
                  conv_cfg=None,
                  norm_cfg=dict(type='BN')):
+        norm_cfg = copy.deepcopy(norm_cfg)
         super().__init__()
         self.in_channels = in_channels
         self.out_channels = out_channels
@@ -157,6 +160,7 @@ class Bottleneck(nn.Module):
                  with_cp=False,
                  conv_cfg=None,
                  norm_cfg=dict(type='BN')):
+        norm_cfg = copy.deepcopy(norm_cfg)
         super().__init__()
         assert style in ['pytorch', 'caffe']
 
@@ -336,6 +340,7 @@ class ResLayer(nn.Sequential):
                  norm_cfg=dict(type='BN'),
                  downsample_first=True,
                  **kwargs):
+        norm_cfg = copy.deepcopy(norm_cfg)
         self.block = block
         self.expansion = get_expansion(block, expansion)
 
@@ -493,6 +498,7 @@ class ResNet(BaseBackbone):
                  norm_eval=False,
                  with_cp=False,
                  zero_init_residual=True):
+        norm_cfg = copy.deepcopy(norm_cfg)
         super().__init__()
         if depth not in self.arch_settings:
             raise KeyError(f'invalid depth {depth} for resnet')

--- a/mmpose/models/backbones/resnet.py
+++ b/mmpose/models/backbones/resnet.py
@@ -44,6 +44,7 @@ class BasicBlock(nn.Module):
                  with_cp=False,
                  conv_cfg=None,
                  norm_cfg=dict(type='BN')):
+        # Protect mutable default arguments
         norm_cfg = copy.deepcopy(norm_cfg)
         super().__init__()
         self.in_channels = in_channels
@@ -160,6 +161,7 @@ class Bottleneck(nn.Module):
                  with_cp=False,
                  conv_cfg=None,
                  norm_cfg=dict(type='BN')):
+        # Protect mutable default arguments
         norm_cfg = copy.deepcopy(norm_cfg)
         super().__init__()
         assert style in ['pytorch', 'caffe']
@@ -340,6 +342,7 @@ class ResLayer(nn.Sequential):
                  norm_cfg=dict(type='BN'),
                  downsample_first=True,
                  **kwargs):
+        # Protect mutable default arguments
         norm_cfg = copy.deepcopy(norm_cfg)
         self.block = block
         self.expansion = get_expansion(block, expansion)
@@ -498,6 +501,7 @@ class ResNet(BaseBackbone):
                  norm_eval=False,
                  with_cp=False,
                  zero_init_residual=True):
+        # Protect mutable default arguments
         norm_cfg = copy.deepcopy(norm_cfg)
         super().__init__()
         if depth not in self.arch_settings:

--- a/mmpose/models/backbones/scnet.py
+++ b/mmpose/models/backbones/scnet.py
@@ -1,3 +1,5 @@
+import copy
+
 import torch
 import torch.nn as nn
 import torch.nn.functional as F
@@ -29,6 +31,7 @@ class SCConv(nn.Module):
                  pooling_r,
                  conv_cfg=None,
                  norm_cfg=dict(type='BN', momentum=0.1)):
+        norm_cfg = copy.deepcopy(norm_cfg)
         super().__init__()
 
         assert in_channels == out_channels

--- a/mmpose/models/backbones/scnet.py
+++ b/mmpose/models/backbones/scnet.py
@@ -31,6 +31,7 @@ class SCConv(nn.Module):
                  pooling_r,
                  conv_cfg=None,
                  norm_cfg=dict(type='BN', momentum=0.1)):
+        # Protect mutable default arguments
         norm_cfg = copy.deepcopy(norm_cfg)
         super().__init__()
 

--- a/mmpose/models/backbones/shufflenet_v1.py
+++ b/mmpose/models/backbones/shufflenet_v1.py
@@ -1,3 +1,4 @@
+import copy
 import logging
 
 import torch
@@ -52,6 +53,8 @@ class ShuffleUnit(nn.Module):
                  norm_cfg=dict(type='BN'),
                  act_cfg=dict(type='ReLU'),
                  with_cp=False):
+        norm_cfg = copy.deepcopy(norm_cfg)
+        act_cfg = copy.deepcopy(act_cfg)
         super().__init__()
         self.in_channels = in_channels
         self.out_channels = out_channels
@@ -184,6 +187,8 @@ class ShuffleNetV1(BaseBackbone):
                  act_cfg=dict(type='ReLU'),
                  norm_eval=False,
                  with_cp=False):
+        norm_cfg = copy.deepcopy(norm_cfg)
+        act_cfg = copy.deepcopy(act_cfg)
         super().__init__()
         self.stage_blocks = [4, 8, 4]
         self.groups = groups

--- a/mmpose/models/backbones/shufflenet_v1.py
+++ b/mmpose/models/backbones/shufflenet_v1.py
@@ -53,6 +53,7 @@ class ShuffleUnit(nn.Module):
                  norm_cfg=dict(type='BN'),
                  act_cfg=dict(type='ReLU'),
                  with_cp=False):
+        # Protect mutable default arguments
         norm_cfg = copy.deepcopy(norm_cfg)
         act_cfg = copy.deepcopy(act_cfg)
         super().__init__()
@@ -187,6 +188,7 @@ class ShuffleNetV1(BaseBackbone):
                  act_cfg=dict(type='ReLU'),
                  norm_eval=False,
                  with_cp=False):
+        # Protect mutable default arguments
         norm_cfg = copy.deepcopy(norm_cfg)
         act_cfg = copy.deepcopy(act_cfg)
         super().__init__()

--- a/mmpose/models/backbones/shufflenet_v2.py
+++ b/mmpose/models/backbones/shufflenet_v2.py
@@ -37,6 +37,7 @@ class InvertedResidual(nn.Module):
                  norm_cfg=dict(type='BN'),
                  act_cfg=dict(type='ReLU'),
                  with_cp=False):
+        # Protect mutable default arguments
         norm_cfg = copy.deepcopy(norm_cfg)
         act_cfg = copy.deepcopy(act_cfg)
         super().__init__()
@@ -162,6 +163,7 @@ class ShuffleNetV2(BaseBackbone):
                  act_cfg=dict(type='ReLU'),
                  norm_eval=False,
                  with_cp=False):
+        # Protect mutable default arguments
         norm_cfg = copy.deepcopy(norm_cfg)
         act_cfg = copy.deepcopy(act_cfg)
         super().__init__()

--- a/mmpose/models/backbones/shufflenet_v2.py
+++ b/mmpose/models/backbones/shufflenet_v2.py
@@ -1,3 +1,4 @@
+import copy
 import logging
 
 import torch
@@ -36,6 +37,8 @@ class InvertedResidual(nn.Module):
                  norm_cfg=dict(type='BN'),
                  act_cfg=dict(type='ReLU'),
                  with_cp=False):
+        norm_cfg = copy.deepcopy(norm_cfg)
+        act_cfg = copy.deepcopy(act_cfg)
         super().__init__()
         self.stride = stride
         self.with_cp = with_cp
@@ -159,6 +162,8 @@ class ShuffleNetV2(BaseBackbone):
                  act_cfg=dict(type='ReLU'),
                  norm_eval=False,
                  with_cp=False):
+        norm_cfg = copy.deepcopy(norm_cfg)
+        act_cfg = copy.deepcopy(act_cfg)
         super().__init__()
         self.stage_blocks = [4, 8, 4]
         for index in out_indices:

--- a/mmpose/models/backbones/utils/inverted_residual.py
+++ b/mmpose/models/backbones/utils/inverted_residual.py
@@ -1,3 +1,5 @@
+import copy
+
 import torch.nn as nn
 import torch.utils.checkpoint as cp
 from mmcv.cnn import ConvModule
@@ -45,6 +47,8 @@ class InvertedResidual(nn.Module):
                  norm_cfg=dict(type='BN'),
                  act_cfg=dict(type='ReLU'),
                  with_cp=False):
+        norm_cfg = copy.deepcopy(norm_cfg)
+        act_cfg = copy.deepcopy(act_cfg)
         super().__init__()
         self.with_res_shortcut = (stride == 1 and in_channels == out_channels)
         assert stride in [1, 2]

--- a/mmpose/models/backbones/utils/inverted_residual.py
+++ b/mmpose/models/backbones/utils/inverted_residual.py
@@ -47,6 +47,7 @@ class InvertedResidual(nn.Module):
                  norm_cfg=dict(type='BN'),
                  act_cfg=dict(type='ReLU'),
                  with_cp=False):
+        # Protect mutable default arguments
         norm_cfg = copy.deepcopy(norm_cfg)
         act_cfg = copy.deepcopy(act_cfg)
         super().__init__()

